gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.html

```diff
@@ -157,6 +161,7 @@
             (click)="onDeletePage.emit(page)"
             [disabled]="page.generalConditions"
             aria-label="Delete page"
+            matTooltip="Delete page"
```

Comment:
Should have a specific tooltip to explain why trashcan is disabled --> `Cannot delete page used by active plans`

Reply:
@jourdiw what about  `Cannot delete page used as general conditions`

Reply:
Yes, with `General Conditions` capitalized. Just make sure that the tooltip message doesn't go onto 2 lines... I had some issues with that for the Unpublish tooltip message when disabled.

Reply:
@jourdiw  
![image](https://github.com/gravitee-io/gravitee-api-management/assets/47851994/4ae82a9f-e956-4611-8dd7-60d32932fd2b)